plugin_1_4.yaml

```diff
@@ -362,11 +408,15 @@ relationships:
         unlink:
           implementation: ansible.cloudify_ansible.tasks.cleanup
 
```

Comment:
We need to keep the old relationships, because there are existing blueprints that use them. I know it's a major plugin upgrade, but we haven't deprecated them formally.